invenio_record_editor/templates/invenio_record_editor/index.html

```diff
@@ -24,16 +24,28 @@
 {%- extends config.RECORDEDITOR_BASE_TEMPLATE -%}
```

Comment:
Let's do this properly from the beginning allowing the template to be configurable. See for example https://github.com/inveniosoftware/invenio-search-ui/search?utf8=%E2%9C%93&q=SEARCH_UI_SEARCH_TEMPLATE

So we will have in `invenio_record_editor/config.py`:

```
RECORD_EDITOR_INDEX_TEMPLATE = 'invenio_record_editor/index.html'
```

The example `index.html` included in this package will just embed the editor in the page.

Then we overwrite that config variable in https://github.com/inspirehep/inspire-next and we add there the code you have added in this PR.